scripts/carrot/HaplotypeCallerComparisonForCarrot.monitoring.wdl

```diff
@@ -0,0 +1,372 @@
+version 1.0
+ 
+ workflow VariantCallingCarrot {
+
+  String pipeline_version = "2.0.3"
+
+  input {
+    Boolean run_dragen_mode_variant_calling = false
+    Boolean use_spanning_event_genotyping = true
+    File calling_interval_list
+    Int haplotype_scatter_count
+    Int break_bands_at_multiples_of
+    Float? contamination = 0
+    File input_bam
+    File input_bam_index
+    File ref_fasta
+    File ref_fasta_index
+    File ref_dict
+    File? ref_str
+    String base_file_name
+    String final_vcf_base_name
+    Int agg_preemptible_tries
+    String gatk_docker = "us.gcr.io/broad-gatk/gatk:4.2.0.0"
+    String gatk_control_docker = "broadinstitute/gatk-nightly:latest"
+    Boolean make_gvcf = true
+    Boolean make_bamout = false
+    Boolean use_gatk3_haplotype_caller = false
+    Boolean skip_reblocking = false
+    Boolean use_dragen_hard_filtering = false
+    File monitoring_script = "gs://emeryj-testing/cromwell_monitoring_script.sh"
```

Comment:
It might not be a bad idea to throw this in a gatk-related bucket or something, but I don't know that it's actually an issue.